WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -45,6 +45,7 @@ import org.wordpress.android.viewmodel.posts.PostListViewModel.PostListEmptyUiSt
 import org.wordpress.android.widgets.RecyclerItemDecoration
 import javax.inject.Inject
 
+private const val EXTRA_POST_LIST_ONLY_USER = "post_list_only_user"
```

Comment:
Can we improve the name so we understand what this constant refers to without checking the implementation for it? Something that refers to the author filter would be great.

Reply:
Change made in 9d1ad99596075800ffeb767c723b6354065e2cb6